Gemfile

```diff
@@ -62,11 +62,11 @@ end
 group :test do
   gem 'database_cleaner'
   gem 'json-schema'
+  gem 'mocha'
```

Comment:
Do these changes actually need mocha and timecop?

Reply:
For Timecop - https://github.com/pieforproviders/pieforproviders/pull/1606#discussion_r705641957

For mocha, I simply moved it up, it was there before. But speaking of it now, maybe we should remove it since it doesn't seem like we are using it

Reply:
Let's leave timecop out for now - RSpec now has native time travel: https://til.hashrocket.com/posts/846d1b9d55-time-travelling-in-rspecrails and as for now we don't need any kind of `tick` behavior with live updates.

As for mocha, I unless it's required by Cypress, which it might be, let's remove it.  But if it's necessary, leave it in.

Reply:
Removed mocha since it seems to not be used